src/instana/instrumentation/tornado/client.py

```diff
@@ -6,25 +6,25 @@
 
     import wrapt
     import functools
-    from typing import TYPE_CHECKING, Dict, Any
```

Comment:
Just noticed this file is not "typed". 
Could you return this line and provide the type annotations we do? If you want, you can do it in a new commit.